reference/docs/conversion/sakai_11_mysql_conversion.sql

```diff
@@ -468,3 +468,12 @@ INSERT INTO SAKAI_REALM_RL_FN VALUES((select REALM_KEY from SAKAI_REALM where RE
 INSERT INTO SAKAI_REALM_RL_FN VALUES((select REALM_KEY from SAKAI_REALM where REALM_ID = '!site.roles'), (select ROLE_KEY from SAKAI_REALM_ROLE where ROLE_NAME = '.default'), (select FUNCTION_KEY from SAKAI_REALM_FUNCTION where FUNCTION_NAME = 'mail.read'));
 INSERT INTO SAKAI_REALM_RL_FN VALUES((select REALM_KEY from SAKAI_REALM where REALM_ID = '!site.roles'), (select ROLE_KEY from SAKAI_REALM_ROLE where ROLE_NAME = '.default'), (select FUNCTION_KEY from SAKAI_REALM_FUNCTION where FUNCTION_NAME = 'rwiki.read'));
 INSERT INTO SAKAI_REALM_RL_FN VALUES((select REALM_KEY from SAKAI_REALM where REALM_ID = '!site.roles'), (select ROLE_KEY from SAKAI_REALM_ROLE where ROLE_NAME = '.default'), (select FUNCTION_KEY from SAKAI_REALM_FUNCTION where FUNCTION_NAME = 'site.visit'));
+
+-- SAK-25099 Anonymous topics in forums
+ALTER TABLE MFR_TOPIC_T ADD COLUMN (POST_ANONYMOUS BIT);
+ALTER TABLE MFR_TOPIC_T MODIFY COLUMN POST_ANONYMOUS BIT NOT NULL;
```

Comment:
On these BIT NOT NULL fields don't we need to provide a default value or provide an update statement like .... 

UPDATE MFR_TOPIC_T SET POST_ANONYMOUS=1 WHERE POST_ANONYMOUS IS NULL;